config/locales/server.en.yml

```
@@ -1,3 +1,7 @@
 en:
+  model:
+    post:
+      delay:
+        error: "You can't post more than once in %{period} in this topic"
```

We should use `dialog` term instead of `topic` in this project.
```suggestion
        error: "You can't post more than once in %{period} in this dialog"
```